tests/testthat/_snaps/chunk-options/chunk-options.Rmd

```diff
@@ -0,0 +1,18 @@
+---
+title: "Untitled"
+author: "M. Salmon"
+date: "September 6, 2018"
+output: html_document
+---
+
+## R Markdown
+
+```{r, message=FALSE}
+#| echo: FALSE
+#| fig.width: 10
+#| fig.cap: This is a long caption.
```

Comment:
maybe that's somewhere where I need to add more YAML-ing as the absence of quote could be problematic?

Reply:
I think it is solved? see the julia chunk below